vehicle/mkvi/software/bms/bms.c

```diff
@@ -225,6 +263,15 @@ int main(void) {
             //         loop_counter = 0;
             //     }
 
+            if (can_poll_receive_dashboard() == 0) {
+                can_receive_dashboard();
+
+                bool ready_to_drive = dashboard.ready_to_drive;
+                if (ready_to_drive) {
+                    control_cooling_loop();
```

Comment:
super nit: based on how you have it configured, we are only going to update the cooling loop PWM when we receive a message from the dashboard - would probably be better to decouple the timing of the systems (ie, in one logic block update the status of `ready_to_drive`, and in another block, implement the cooling loop logic based on the most recevent value received from the dashboard